### PR TITLE
Added a reminder that Account Admin permission was required to the Ge…

### PR DESCRIPTION
**Pull request by gshannon-immersve**
…t, Create, and List Funding Channel api-doc pages.

Your description here.

Ticket Link: _link_here_

### CODE REVIEW

**Review by NicBathgate — APPROVED** [review]
Love this, very helpful information.

Subjective feedback: Might read nicer to say "Note: Account Admin..." instead of "Remember that Account Admin..."

File: imsv-docs-docusaurus/openapi/endpoints/funding-channel/funding-channel-create.yaml [hunk 1]
```diff
@@ -6,6 +6,8 @@ post:
   description: |
     This endpoint can be used to create a Funding Channel.
     See [Funding Protocols](https://docs.immersve.com/guides/funding-protocols/) for available funding types.
+
+    **Remember that Account Admin permission is required to access this endpoint.**
```
Comment (gshannon-immersve) [inline imsv-docs-docusaurus/openapi/endpoints/funding-channel/funding-channel-create.yaml:10]:
```suggestion
    **Note: Account Admin permission is required to access this endpoint.**
```

File: imsv-docs-docusaurus/openapi/endpoints/funding-channel/funding-channel-get.yaml [hunk 2]
```diff
@@ -3,7 +3,10 @@ get:
     - funding-channel
   summary: Get Funding Channel
   operationId: get-funding-channel
-  description: Retrieves a specific Funding Channel for the given id.
+  description: |
+    Retrieves a specific Funding Channel for the given id.
+
+    **Remember that Account Admin permission is required to access this endpoint.**
```
Comment (gshannon-immersve) [inline imsv-docs-docusaurus/openapi/endpoints/funding-channel/funding-channel-get.yaml:9]:
```suggestion
    **Note: Account Admin permission is required to access this endpoint.**
```

File: imsv-docs-docusaurus/openapi/endpoints/funding-channel/funding-channels-list.yaml [hunk 3]
```diff
@@ -7,6 +7,8 @@ get:
     Retrieves a list of Funding Channels for the given account
 
     This API uses cursor-based pagination. Start by making a request without a cursor to get the first page. Use the nextCursor from the pageInfo in the response as the cursor for subsequent requests to retrieve further pages. Continue until nextCursor is undefined, indicating no more data.
+
+    **Remember that Account Admin permission is required to access this endpoint.**
```
Comment (gshannon-immersve) [inline imsv-docs-docusaurus/openapi/endpoints/funding-channel/funding-channels-list.yaml:11]:
```suggestion
    **Note: Account Admin permission is required to access this endpoint.**
```

**Comment by gshannon-immersve** [comment]
closing this, as in discussion with @ncjones, we don't want tight coupling between endpoints and permissions roles (and thus don't want to document that it is currently this way)

**Pull request closed without merging**